API-Automation/test-suites/pet/get_request.spec.js

```diff
@@ -0,0 +1,19 @@
+import { findPetById, findPetByStatus } from "../../tasks/api-tasks.js";
+
+
+describe('PET - GET API - positive', () => {
+
+    it('TC_10_1 User is able to find the pet by status', async () => {
```

Comment:
Please check beforeAll() and beforeEach() functions in Jasmine. Here, you should create one pet before these two its(), and then try to get the created one with different endpoints.